packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -31,16 +31,100 @@
 
         <div v-else-if="error">{{ t('users.load_list_error') }}</div>
 
-        <UsersTable
-          v-else-if="usersList.length >= 1"
-          :usersList="usersList"
-          :fetching="fetching"
-          :error="error"
-          @goToUserDetails="goToUserDetails"
-          @makeUserAdmin="makeUserAdmin"
-          @makeAdminToUser="makeAdminToUser"
-          @deleteUser="deleteUser"
-        />
+        <div v-else-if="usersList.length >= 1" class="m-5">
```

Comment:
Instead of the `>= 1` condition, let's use the `>0` check, hence it'll be consistent across the platform.